mac/Keyman4MacIM/Keyman4MacIM/Info.plist

```diff
@@ -36,7 +36,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>Keyman $(PRODUCT_VERSION) for macOS</string>
+	<string>14.0.184</string>
```

Comment:
Don't we want the auto-generated version?

Reply:
It doesn't really matter because it is overwritten during the build. There is a TODO to have the build rewrite the plists post-build in the generated artifacts rather than pre-build so that we don't continually have them stomped when working locally. I hope to do that shortly (separate PR though).

Reply:
Separate PR is now up at #3891.